pkgs/development/compilers/llvm/7/libc++abi.nix

```diff
@@ -1,4 +1,5 @@
 { stdenv, cmake, fetch, libcxx, llvm, version
+, standalone ? false
```

Comment:
This has very similar effects to `stdenv.hostPlatform.useLLVM`, in supporting bootstraping an llvm/clang based toolchain. Though useLLVM is only defined in versions 8, 9, 10 and 11. We should find a consistent approach to this. If possible I'd like to remove `useLLVM` from `llvm/*/libc++abi.nix` for something reusable that describes its effects.